src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicSpecifyCultureForToLowerAndToUpper.vb

```diff
@@ -0,0 +1,25 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+Imports Microsoft.NetCore.Analyzers.Runtime
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Public Class BasicSpecifyCultureForToLowerAndToUpperAnalyzer
+        Inherits SpecifyCultureForToLowerAndToUpperAnalyzer
+
+        Protected Overrides Function GetMethodNameLocation(invocationNode As SyntaxNode) As Location
+            Debug.Assert(invocationNode.IsKind(SyntaxKind.InvocationExpression))
+
+            Dim invocation = CType(invocationNode, InvocationExpressionSyntax)
```

Comment:
This can fail I think. VB doesn't require parentheses for parameterless method calls

```vb
Public Class C
    Public Sub M(x As String)
        Dim y = x.ToLower ' This is a member access expression, not an invocation.
    End Sub
End Class
```